interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1638,6 +1637,14 @@ namespace cling {
       ->addCallback(std::move(C));
   }
 
+  const DynamicLibraryManager* Interpreter::getDynamicLibraryManager() const {
+    return const_cast<Interpreter*>(this)->getDynamicLibraryManager();
+  }
+  DynamicLibraryManager* Interpreter::getDynamicLibraryManager() {
+    assert(m_Executor.get() && "We must have an executor");
+    return &m_Executor->getDynamicLibraryManager();
```

Comment:
...forward to `const_cast<const Interpreter*>(this)->getDynamicLibraryManager()` here. I.e. better to add than remove const.

Reply:
Looks like the pattern in the llvm is the inverse. I'd keep this as is.

Reply:
Rationale of my suggestion: it's safer to add than to remove `const`. And I see plenty of examples adding `const` in llvm, see e.g. `DenseMap.h` so I wouldn't call it a pattern in llvm?
```
  BucketT *getInlineBuckets() {
    return const_cast<BucketT *>(
      const_cast<const SmallDenseMap *>(this)->getInlineBuckets());
  }
```

Reply:
Fair enough.